assets/app/views/_overview-deployment.html

```diff
@@ -49,7 +49,7 @@ <h3 class="h4">
           <span class="deployment-trigger" ng-repeat="cause in rc.causes">
             <span ng-switch="cause.type">
               <span ng-switch-when="ImageChange">
-                a new image <span ng-if="cause.imageTrigger.from">for {{cause.imageTrigger.from | imageObjectRef : null : true}}</span>
```

Comment:
Hm, yeah, that example causes a problem with the change.

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/8549444/c7da8dcc-2498-11e5-8002-0f2efbc5971a.png)